site/content/en/docs/Guides/fleet-updates.md

```diff
@@ -105,4 +105,11 @@ Fleets through that mechanism. The `preferred` label matching selector tells the
 all `GameServers` with the `v2` `Fleet` label, and if not found, search through the rest of the set.
 
 The above `GameServerAllocation` can then be used while you scale up the `v2` Fleet and scale down the original Fleet at
-the rate that you deem fit for your specific rollout. 
\ No newline at end of file
+the rate that you deem fit for your specific rollout.
+
+## Alpha Feature RollingUpdateOnReady
+
+{{< alpha title="Rolling Update on Ready" gate="RollingUpdateOnReady" >}}
+
+If we are updating the Fleet configuration, the new GameServerSet would be created with 0 GameServers at the beginning, if RollingUpdate deployment strategy is used. After creating a first batch of `MaxSurge` GameServers, old GameServerSet should be waiting before some of them become Ready, before scaling down GameServers which belong to an old GameServerSet.
```

Comment:
This feels to me like it's written in the negative. I would recommend writing it in the postive.

Something like:

> When this feature is enabled, Fleets will wait for the new GameSevers to become Ready during a Rolling Update, to ensure there is always a set of Ready GameServers before attempting to shut down the previous version Fleet's GameServers

> This ensures a Fleet cannot accidentally have 0 GameServers Ready if something goes wrong during a RollingUpdate, or GameServers have a long delay when moving to a Ready state.

What do you think of that?

Reply:
Thanks for an advice. I will rewrite it today.